dev_guide/deployments/advanced_deployment_strategies.adoc

```diff
@@ -238,27 +238,46 @@ spec:
 ...
 ----
 
+[[manaing-weights-using-the-web-console]]
+==== Managing Weights Using the Web Console
+
+. Navigate to the Route details page (Applications/Routes).
+. Select *Edit* from the Actions menu.
+. Check *Split traffic across multiple services*.
+. The *Service Weights* slider sets the percentage of traffic sent to each service.
++
+image::3.7_edit_route.png[Service Weights]
++
+For traffic split between more than two services, the relative weights are
+specified by integers between 0 and 256 for each service.
++
+image::3.7_weighting_integers.png[Weights specified by integers]
++
+Traffic weightings are shown on the *Overview* in the expanded rows of the
+applications between which traffic is split.
++
+image::weighting.png[Weighting in the Web Console]
```

Comment:
FYI, you have this same image twice. Once here, and once above the Managing Weights section